src/utils/daos.c

```diff
@@ -1280,6 +1382,19 @@ help_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 
 	if (argc <= 2) {
 		FIRST_LEVEL_HELP();
```

Comment:
In the definition for `FIRST_LEVEL_HELP` (above on line ~1300), we should have an entry for filesystem (fs).
This will add an entry for `daos help`

Reply:
@daltonbohning added this in latest commit